File: src/main/java/pl/kedziorek/silka/silownia/model/Sprzet.java
package pl.kedziorek.silka.silownia.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "SPRZET")
@Getter
@Setter
@NoArgsConstructor

public class Sprzet {
    @Id
    @Column(name ="ID")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private  Long sprzetId;

    @Column(name ="NAZWA")
    private String sprzetNazwa;

    @Column(name ="KATEGORIA")
    private Integer sprzetKategoria;

    @Column(name ="STATUS")
    private Integer sprzetStatus;

    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "SILOWNIAID")
    private Silownia silownia;
}

File: silownia-app/src/app/sprzet/sprzet.ts
import {Silownia} from "./silownia";

export class Sprzet {
  sprzetId: number;
  sprzetNazwa: string;
  sprzetKategoria: number;
  sprzetStatus: number;
  silownia: Silownia;

  constructor(sprzetId: number, sprzetNazwa: string, sprzetKategoria: number, sprzetStatus: number, silownia: Silownia){
    this.sprzetId = sprzetId;
    this.sprzetNazwa = sprzetNazwa;
    this.sprzetKategoria = sprzetKategoria;
    this.sprzetStatus = sprzetStatus;
    this.silownia= silownia;
  }
}

